packages/contracts/contracts/strategies/HackMoneyStrategy.sol

```diff
@@ -505,6 +515,7 @@ contract HackMoneyStrategy is HackMoneyStrategyBase, IHackMoneyStrategy {
             secondsToExpiry <= strategyDetail.maxTimeToExpiry);
     }
 
+    // TODO: Think how to handle dynamic fees of synthetix not to sell cheap.. https://blog.synthetix.io/dynamic-exchange-fees-explained/
```

Comment:
I dont think this is an issue, we use Synthetix to exchange premiums. Premiums are small portion of the notional and fees on exchanging premiums is a small portion of premiums, a max of 0.3% rare difference on premiums will not hurt in my opinion. Besides we are trading eth/usd which is not subject to high volatility compared to other markets. However Ill check, maybe we set a max fee to accept say 0.25%?

Reply:
Synthetix have 0 slippage, see https://docs.synthetix.io/litepaper/#system-architecture (cmd+F 'slippage'), only trading fees are paid.

Reply:
Yep, but that's what I refer. We can have slippage that would be reflecting fees that we have paid